src/guc.c

```diff
@@ -1011,6 +1019,30 @@ _guc_init(void)
 							 NULL,
 							 NULL);
 
+	DefineCustomEnumVariable(MAKE_EXTOPTION("compress_truncate_behaviour"),
+							 "Define behaviour of truncate after compression",
+							 "Define behaviour of truncate after compression",
+							 (int *) &ts_guc_compress_truncate_behaviour,
+							 COMPRESS_TRUNCATE_ONLY,
+							 compress_truncate_behaviour_options,
+							 PGC_USERSET,
+							 0,
+							 NULL,
+							 NULL,
+							 NULL);
+
+	DefineCustomBoolVariable(MAKE_EXTOPTION("enable_recompress_waiting"),
+							 "enable recompress spin lock",
+							 "enable recompress spin lock",
```

Comment:
Same here, please leverage the long description for additional context. Thank you